python/cuml/dask/common/dask_arr_utils.py

```diff
@@ -53,6 +68,31 @@ def _conv_df_to_sp(x):
     return cp.sparse.csr_matrix(cupy_ary)
 
 
+def _conv_to_sparse(arr):
+
```

Comment:
stray blank line?

Reply:
would be nice to have a quick one-liner docstring here - was not immediately obvious that this always returns cupy sparse even when given scipy sparse e.g.